lib/ansible/plugins/connection/sshchroot.py

```diff
@@ -0,0 +1,227 @@
+# Copyright: (c) 2018, Anthony ARNAUD <github@anthony-arnaud.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+from ansible.errors import AnsibleError
+from ansible.plugins.connection.ssh import Connection as SSHConnection
+from ansible import constants as C
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    connection: sshchroot
+    short_description: connect via ssh client binary
+    description:
+        - This connection plugin allows ansible to communicate to the target machines via normal ssh command line.
+    author: ansible (@core)
+    version_added: historical
```

Comment:
This should reflect `2.8` as we're currently in the [Ansible 2.8 development cycle](https://docs.ansible.com/ansible/devel/roadmap/).